### PR TITLE
[improvement][config] Update datasource.properties，add mysql meta data template

### PR DESCRIPTION
**Pull request by Jatham922**
[新增]元数据mysql连接模版

## *Tips*
- *Thanks very much for contributing to Apache DolphinScheduler.*
- *Please review https://dolphinscheduler.apache.org/en-us/community/index.html before opening a pull request.*

## What is the purpose of the pull request

*(For example: This pull request adds checkstyle plugin.)*

## Brief change log

*(for example:)*
  - *Add maven-checkstyle-plugin to root pom.xml*

## Verify this pull request

*(Please pick either of the following options)*

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

*(example:)*

  - *Added dolphinscheduler-dao tests for end-to-end.*
  - *Added CronUtilsTest to verify the change.*
  - *Manually verified the change by testing locally.*

### CODE REVIEW

**Comment by Jatham922** [comment]
新增mysql元数据连接模版

**Review by CalvinKirs — COMMENTED** [review]
It seems that there is no problem. But can we do this? @lgcareer

**Comment by Jatham922** [comment]
I think it is convenient for mysql users who is junior programmer.

File: dolphinscheduler-dao/src/main/resources/datasource.properties [hunk 1]
```diff
@@ -21,6 +21,13 @@ spring.datasource.url=jdbc:postgresql://localhost:5432/dolphinscheduler
 spring.datasource.username=test
 spring.datasource.password=test
 
+# mysql
+#spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
```
Comment (chengshiwen) [inline dolphinscheduler-dao/src/main/resources/datasource.properties:25]: Require `spring.datasource.driver-class-name=com.mysql.jdbc.Driver` in `mysql-connector-java 5`

Comment (Jatham922) [reply dolphinscheduler-dao/src/main/resources/datasource.properties:25]: Thx

**Review by dailidong — APPROVED** [review]
+1
good job

**Comment by dailidong** [comment]
> It seems that there is no problem. But can we do this? @lgcareer

we can do this

File: dolphinscheduler-dao/src/main/resources/datasource.properties (hunk 1 above)
Comment (dailidong) [reply dolphinscheduler-dao/src/main/resources/datasource.properties]: > Require `spring.datasource.driver-class-name=com.mysql.jdbc.Driver` in `mysql-connector-java 5`

I have changed this config

**Comment by dailidong** [comment]
Thanks for your first contribution,  please describe PR in English next time

**Pull request merged**